lib/primer/forms/primer_text_field.ts

```diff
@@ -97,11 +97,19 @@ export class PrimerTextFieldElement extends HTMLElement {
 
   showLeadingSpinner(): void {
     this.leadingSpinner?.removeAttribute('hidden')
+    const spinnerScreenreaderText = this.leadingSpinner?.nextElementSibling
```

Comment:
Same question re: using a Catalyst target for this. Then we could do `this.spinnerScreenreaderText` or similar.

Reply:
Can that be referenced from the text field TS file, or will I need to set up a new corresponding TS file for the spinner component? The reasons I reverted to this class-based solution (besides being new to Catalyst) were 1) I wasn't sure if it'd be overkill to set up an entirely new file just for this screen reader only text, and 2) logging the `leadingSpinner`object only seemed to report the content inside the `svg` tag (the offending text is a sibling instead of a child), so I was unsure if setting Catalyst up for the spinner would be less convoluted than this.

Reply:
Ahh ok I totally missed these changes were in the `Spinner` and `PrimerTextField` code. I agree it feels like overkill to define a custom element for `Spinner` just to expose a single DOM element.

What you've got here seems ok then. Although `nextElementSibling` makes me a little nervous 😅 Maybe we can do `this.leadingSpinner?.querySelector('.spinner-screenreader-text)?`.